src/components/ui/Button/buttons.scss

```diff
@@ -1,23 +1,34 @@
 @import "../../../styles/utilities";
 
 .button[data-store-button] {
-  display: flex;
+  display: inline-flex;
   align-items: center;
   justify-content: center;
+  min-height: var(--min-size);
   padding: var(--space-1)  var(--space-3);
   font-weight: var(--text-weight-bold);
+  text-decoration: none;
   column-gap: var(--space-2);
   border: 0;
   border-radius: var(--border-radius-default);
   transition: background-color .5s ease;
 
   &:focus { @include focus-ring; }
 
-  &:disabled {
+  &:disabled, &[data-button-disabled="true"] {
     cursor: not-allowed;
+    pointer-events: none;
     svg { color: var(--color-neutral-5); }
   }
 
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:visited {
```

Comment:
Maybe we can separate this a bit to make it less confusing.

Does it make sense to add a `button-link` class on this component just to toggle link specif attributes? 🤔